client/main.less

```diff
@@ -1,5 +1,5 @@
-@phone: ~"(max-width: 767px)";
-@tablet: ~"(min-width: 767px) and (max-width: 991px)";
+@phone: ~"(max-width: 360px)";
+@tablet: ~"(min-width: 361px) and (max-width: 991px)";
```

Comment:
Rozdzielczość fizyczna != logiczna
Co z tego, że twój telefon ma 1440px szerokości, skoro dla przeglądarki będzie to ok 360px